client/src/components/RhythmCodeIcon.js

```diff
@@ -10,9 +10,9 @@ function RhythmCodeIcon({
 	rhythmsList,
 	setModalOpen,
 	setSelectedRhythm,
+	toggleDropdownShown,
+	isOn,
 }) {
-	const [isDropdownShown, setDropdownShown] = useState(false);
```

Comment:
I don't mind replacing the state here with the solution in place, but I'd like to keep the explicit naming convention.

Reply:
name fixed